static/frontpage/frontpage.html

```diff
@@ -318,13 +318,7 @@ <h3><a href="https://www.linkedin.com/in/murat-semerci-phd-911a722/">Murat Semer
       <p>PANION / ABB eMobility Digital Venture Gmbh'de Kıdemli Veri Bilimci<br><i>Baş Çevirmen</i></p>
     </div>
   </div>
-  <div class = "mdl-cell mdl-cell--3-col mdl-cell--top">
-    <div class="author-item">
-      <img src="./_images/baris.jpg"/>
-      <h3><a href="https://www.linkedin.com/in/baris-yasin/">Barış Yaşin</a></h3>
-      <p>Amazon'da Çözüm Mimari<br><i>Baş Editör</i></p>
-    </div>
-  </div>
+
```

Comment:
@SemerciMurat @astonzhang Just want to make sure this change is intentional or made by mistake?

Reply:
Hi @astonzhang  and @AnirudhDagar ,

As mentioned in the meeting before, @barisyasin, he had told that he would not have time to focus on the translation projects in the future due to his work load. He has not been active almost for a year. I made this change but we keep him in the Turkish preface page (at the last paragraph) as a Translator contributor (Türkçe çevirisindeki katkılarından dolayı Murat Semerci ve Barış Yaşin’e teşekkür ediyoruz.) since he had some good contribution at the beginning of the translation project.

Reply:
I see, thanks for the clarification.